src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -36,7 +48,10 @@ const StyledCheckBoxContainer = styled.label`
   flex-direction: row;
   align-items: center;
   user-select: none;
-  width: fit-content;
+  ${props =>
+    props.fill
+      ? fillStyle(props.fill) && `justify-content: space-between;`
```

Comment:
```suggestion
      ? fillStyle() && `justify-content: space-between;`
```
Since `fillStyle` will only be applied when `props.fill` is true, I think we can simplify the function to not have any  parameters.